client/asset/btc/btc.go

```diff
@@ -1824,9 +1824,10 @@ func (btc *ExchangeWallet) SignMessage(coin asset.Coin, msg dex.Bytes) (pubkeys,
 	return
 }
 
-// AuditContract retrieves information about a swap contract on the blockchain.
-// AuditContract would be used to audit the counter-party's contract during a
-// swap.
+// AuditContract retrieves information about a swap contract from the provided
+// txData. The extracted information would be used to audit the counter-party's
+// contract during a swap.
+// TODO: Broadcast the txData.
```

Comment:
How about we do this now with the following right before `return`:

```go
hashSent, err := btc.node.sendRawTransaction(tx)
if err != nil {
      btc.log.Debugf("Rebroadcasting counterparty contract %v (THIS MAY BE NORMAL): %v", txHash, err)
} else if !hashSent.IsEqual(txHash) {
       btc.log.Errorf("Counterparty contract %v was rebroadcast as %v!", txHash, hashSent)
}
```